### ISSUE TITLE
`.woff2` version

### ISSUE THREAD

**Issue by o-t-w** [justvanrossum/nabla#57]
Is there a `.woff2` version you could include in the download? I know there are tools I could use to convert it but some of those are not compatible with variable color fonts afaik.

**Comment by anthrotype** [comment]
> some of those are not compatible with variable color fonts afaik

I don't think so. What do you mean by that? woff2 compressors (e.g. the c++ one from google/woff2 repo, or the `fonttools ttLib.woff2 compress` script in the python [FontTools](https://github.com/fonttools/fonttools) library) do not care whether the font contains variation tables or has COLR or SVG or any other color tables.

**Comment by o-t-w** [comment]
Last time I checked the webfont generator at [FontSquirrel](https://www.fontsquirrel.com/tools/webfont-generator) messes them up.

**Comment by anthrotype** [comment]
maybe they are dropping tables they don't know about, I have no idea how that works, but the mere act of woff2-compressing a font does not involve dealing with variable or color tables at all.
I suggest you try using woff2_compress command line tool directly https://github.com/google/woff2

**Comment by justvanrossum** [comment]
I wonder why the download from Google Fonts doesn't include `.woff2`.

**Comment by justvanrossum** [comment]
I just checked, and FontSquirrel indeed drops the COLR/CPAL tables.
